### PR TITLE
Update submission table status

### PR DESCRIPTION
**Pull request by jabrah**
#568

### CODE REVIEW

File: app/templates/components/submission-status-cell.hbs [hunk 1]
```diff
@@ -19,33 +20,34 @@
 </style>
 
 <div class="text-left">
-    {{#if (eq record.source 'other')}}
-      {{#if (eq record.submitted true)}}
-      <span class="status accepted pr-1">
+  <span class="status {{status}} pr-1">
+    {{#if (eq status "Complete")}}
+      {{#if (eq record.source "other")}}
         <span tooltip-position="left" tooltip="Funder has receive a submission for this publication from a different source.">
```
Comment (karenhanson) [inline app/templates/components/submission-status-cell.hbs]: This should be "All target repositories have received a copy of the Submission and verified that it is approved for publication"

File: app/templates/components/submission-status-cell.hbs [hunk 2]
```diff
@@ -19,33 +20,34 @@
...
-        {{#if (eq status "Not Started")}}
+    {{/if}}
+    {{#if (eq status "Manuscript expected")}}
+      <span tooltip-position="left" tooltip="Your funder is aware of this publication and is expecting the deposit of your manuscript.">
+        <i class="fas fa-info-circle"></i> {{status}}
+      </span>
+    {{/if}}
+    {{#if (eq status "Submitted")}}
+      {{#if (eq record.source "pass")}}
         <span tooltip-position="left" tooltip="This submission is in queue to be transmitted to target its repositories.">
```
Comment (karenhanson) [inline app/templates/components/submission-status-cell.hbs]: Change to "This submission has either been sent to all target repositories, or is in the queue and will be submitted shortly"

Comment (htpvu) [reply app/templates/components/submission-status-cell.hbs]: this looks good.

File: app/templates/components/submission-status-cell.hbs (hunk 1 above)
Comment (htpvu) [reply app/templates/components/submission-status-cell.hbs]: this looks fine too!

Comment (jabrah) [reply app/templates/components/submission-status-cell.hbs]: Should `Complete` status care about the submission source?

**Pull request merged**